CONTRIBUTING.md

```diff
@@ -66,7 +66,7 @@ If you observe following error running any of the test files that means you are
 
 ```sh
 cd packages/beacon-node
-LODESTAR_PRESET=minimal yarn vitest --run --bail 1 --config vitest.spec.config.ts test/spec/presets/sanity.test.ts -t attester_slashing
```

Comment:
just to confirm, if I run this command in a package directory (e.g. packages/beacon-node) will it load the environment variables in .env.test?

Reply:
Yes